### PR TITLE
Commit for working on RT for the workflow (jenkins)

### PR DESCRIPTION
**Pull request by pruzanov**

### CODE REVIEW

**Comment by pruzanov** [comment]
retest this please

File: variantMerging.wdl [hunk 1]
```diff
@@ -97,39 +139,64 @@ runtime {
...
+  javaMemory = ~{jobMemory} - 6 
+  gatkCommand  = "$JAVA_ROOT/bin/java -Xmx" + str(javaMemory) + "G -jar $GATK_ROOT/GenomeAnalysisTK.jar "
+  gatkCommand += "-T CombineVariants "
+  gatkCommand += " ".join(inputStrings)
+  gatkCommand += " -R ~{referenceFasta} "
+  gatkCommand += "-o ~{outputPrefix}_combined.vcf.gz "
+  gatkCommand += "-genotypeMergeOptions PRIORITIZE "
+  gatkCommand += "-priority " + "~{sep=',' workflows}"
+
+  res_string = os.popen(gatkCommand + " 2>&1").read()
```
Comment (mlaszloffy) [inline variantMerging.wdl:197]: The exit code of popen isn't being checked, maybe switch from "os" to "subprocess".

Comment (pruzanov) [reply variantMerging.wdl]: refactored, changed to subrocess

File: variantMerging.wdl [hunk 2]
```diff
@@ -97,39 +139,70 @@ runtime {
...
+  priority = p.split()
+  
+  if len(vcfFiles) != len(workflowIds):
+      print("The arrays with input files and their respective workflow names are not of equal size!")
+  else:
+      for f in range(0, len(vcfFiles)):
+          inputStrings.append("--variant:" + workflowIds[f] + " " + vcfFiles[f])
+
+  javaMemory = ~{jobMemory} - 6 
+  gatkCommand  = "$JAVA_ROOT/bin/java -Xmx" + str(javaMemory) + "G -jar $GATK_ROOT/GenomeAnalysisTK.jar "
```
Comment (mlaszloffy) [inline variantMerging.wdl:194]: "$JAVA_ROOT/bin/java" and "$GATK_ROOT/GenomeAnalysisTK.jar" probably should be parameterized as a WDL parameter.

**Review by lheisler — APPROVED** [review]
there's extensive work here; i've done a cursory review to move this forward

**Pull request merged**